kubetest/kind/kind.go

```diff
@@ -272,7 +281,7 @@ func (d *Deployer) Build() error {
 		cmd := exec.Command(
 			"bazel", "build",
 			"//cmd/kubectl", "//test/e2e:e2e.test",
-			"//vendor/github.com/onsi/ginkgo/ginkgo",
+			"//"+ginkgoTarget,
```

Comment:
don't template this, ginkgo can't be in different places in bazel, bazel was removed well before v2.

Reply:
agree, `ginkgoTarget` is set to `vendor/github.com/onsi/ginkgo/ginkgo` for bazel because the same reason.